packages/code-du-travail-api/src/server/routes/search/utils.js

```diff
@@ -46,6 +48,19 @@ const mergePipe = (a, b, max_result) => {
   return mergeDuplicate(res).slice(0, max_result);
 };
 
+const getDataFromUrl = (url) => {
+  const [, sourceRoute, slug] = url.split("/");
+  let source = getSourceByRoute(sourceRoute);
+  if (source === SOURCES.SHEET_MT && !slug.includes("#")) {
```

Comment:
Does this could lead to problem when there is anchorless sections ?

Reply:
Anchorless sections are introductions so there is no problem with that since there will always be at the top of the page

Reply:
(the content in the tiles is the same than the full pages)

Reply:
it fix the problem of invisible display, but to me the problem that we should fix, is why introduction document are not found :/

Reply:
We have the same url for both content so this is inevitable (I think). We have to make a choice when we resolve this. The only other solution I can see is to put the source in the ref but that's what we are already doing in the admin so I guess there's no use in making a refactor right now for something that will "auto-fix" in a near future

Reply:
https://github.com/SocialGouv/code-du-travail-numerique/blob/master/packages/sources/index.js#L34

Reply:
(it's not really that we don't find them it's that we decided that we won't put any partial fiches MR in the refs for now form the datafiller or the admin, so they are not supposed to any SOURCES.SHEET_MT in the future, but they do still are here, because data still has some "#" in refs). And if the fiches were empty, it's not because we don't find them, it's because we had a difference in the ref resolver's logic

Reply:
the only mecanism which uses the partial fiches MT now is in fact the user's search. Otherwise we deal with entire pages.